docs/language/control-flow.md

```diff
@@ -126,8 +125,9 @@ executed if the condition is true inside opening and closing braces.
 The condition must be boolean and the braces are required.
 
 The while-statement will first evaluate the condition.
-If the condition is false, the execution is done.
 If it is true, the piece of code is executed and the evaluation of the condition is repeated.
+If the condition is false, the piece of code is not executed and the execution
+of the whole while-statement is finished.
```

Comment:
Nice improvement! Minor: We normally break on logical sentence breaks (e.g. conjunctions)

```suggestion
If the condition is false, the piece of code is not executed 
and the execution of the whole while-statement is finished.
```